f2/src/utils/formatAnalystEmail.js

```diff
@@ -66,7 +66,7 @@ const formatVictimDetails = (data) => {
   let postalProv = ''
   try {
     let location = zipcodes.lookup(data.location.postalCode)
-    if (location === undefined) {
+    if (data.location.postalCode && location === undefined) {
```

Comment:
this cause error in console 'TypeError: Cannot read property 'city' of undefined' 
now with the new condition, the logic try to fetch location.city when it is anonymous report.